.github/CHANGELOG.md

```diff
@@ -26,6 +26,48 @@
 
 * Ising YY gate functionality added.
   [(#1358)](https://github.com/PennyLaneAI/pennylane/pull/1358)
+  
+* Added functionality to `qml.sample()` to extract samples from the basis states of
+the device (currently only for qubit devices). Additionally, `wires` can be
+specified to only return samples from those wires. 
+[(#1441)](https://github.com/PennyLaneAI/pennylane/pull/1441)
+
+```python
+dev = qml.device("default.qubit", wires=3, shots=5)
+
+@qml.qnode(dev)
+def circuit_1():
+    qml.Hadamard(wires=0)
+    qml.Hadamard(wires=1)
+    return qml.sample()    # no observable provided
+
+@qml.qnode(dev)
+def circuit_2():
+    qml.Hadamard(wires=0)
+    qml.Hadamard(wires=1)
+    return qml.sample(wires=[0,2])    # no observable provided and wires specified 
+``` 
```

Comment:
Same here - this entire codeblock (including the backticks) needs to be indented by two spaces